client/user/src/mealorder/menu/meal-order-drawer.html

```diff
@@ -0,0 +1,201 @@
+  ~
+  ~ THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+  ~ "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+  ~ LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+  ~ A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+  ~ OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+  ~ SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+  ~ LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+  ~ DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+  ~ THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+  ~ (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+  ~ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+  -->
+<link rel="import" href="../../../bower_components/app-layout/app-toolbar/app-toolbar.html">
+<link rel="import" href="../../../bower_components/app-route/app-location.html">
+<link rel="import" href="../../../bower_components/app-route/app-route.html">
+<link rel="import" href="../../../bower_components/iron-pages/iron-pages.html">
+<link rel="import" href="../../../bower_components/app-datepicker/app-datepicker.html">
+
+<dom-module id="meal-order-drawer">
+    <template>
+        <style>
+            .app-toolbar-header {
+                display: flex;
+                justify-content: space-between;
+                align-items: center;
+                width: 100%;
+                z-index: 1;
+            }
+
+            .app-toolbar-header .app-toolbar-content {
+                font-size: 24px;
+            }
+
+            #day {
+                font-size: 0.7em;
+                font-weight: lighter;
+                text-align: center;
+            }
+
+            #day-of-the-week {
+                font-size: 22px;
+            }
+
+            #date-picker {
+                z-index: 0;
+                position: absolute;
+                top: 84px;
+
+                --app-datepicker-selection-bg: var(--paper-grey-50);
+                --app-datepicker-selection-color: var(--google-grey-700);
+                --app-datepicker-iron-selected: var(--google-grey-700);
+                --app-datepicker-selected-day-bg: var(--paper-light-blue-500);
+                --app-datepicker-today-color: var(--paper-light-blue-500);
+                --app-datepicker-date-hover-background-color: var(--paper-light-blue-500);
+
+                --app-datepicker-selected-date: {
+                    display: none;
+                };
+
+                --app-datepicker-selected-year: {
+                    display: none;
+                };
+            }
+
+        </style>
+        <app-location route="{{route}}"
+                      use-hash-as-path></app-location>
+
+        <app-route
+                route="{{route}}"
+                pattern="/:page/:date"
+                data="{{routeData}}">
+        </app-route>
```

Comment:
This is done in . <app> component.

Reply:
Can you please describe in more detail?

Reply:
Is it good practice to change routing by two way binding in this case?